services/ui-src/src/index.scss

```diff
@@ -67,6 +67,16 @@ $color-primary-alt-lightest: #f0fafe;
     font-weight: bold;
   }
 
+  .status-bandage {
+    // the min and max allow for minor growing/shrinking
+    // to condense the width of the column when scaled down
+    // At full screen, the extra 25px lets the status breathe
+    max-width: 170px;
+    min-width: 145px;
+    // Responsible for wrapping any column
+    white-space: normal;
+  }
+
```

Comment:
This works though on large screens still wraps when unnecessary.  Having said that I can also see the wrapping at a fixed size making sense too so that a single column doesnt push the rest too far to the right. Just a note, nothing to really fix here if this was intentional.

Reply:
This behavior addresses AC number 1: 
> The status column should be sized reasonably in comparison to the table size and other columns

No official size was given, so I used my better judgement! I did update the method re: the above comment, so now technically the _column_ doesn't have a max width, just the `p` element inside, and the cell will adjust width accordingly.